.storybook/constants.ts

```diff
@@ -14,21 +15,7 @@ const i18nData: AppProps['i18nData'] = {
   localeMessages: englishMessages,
 };
 
-const nodeVersionData: NodeVersionData[] = [
-  {
-    node: 'v19.8.1',
-    nodeNumeric: '19.8.1',
-    nodeMajor: 'v19.x',
-    npm: '9.5.1',
-    isLts: false,
-  },
-  {
-    node: 'v18.15.0',
-    nodeNumeric: '18.15.0',
-    nodeMajor: 'v18.x',
-    npm: '9.5.0',
-    isLts: true,
-  },
-];
-
-export const pageProps = { i18nData, nodeVersionData };
+export const pageProps = {
+  i18nData,
+  nodeReleasesData: createNodeReleasesData(),
```

Comment:
Assign this to a variable just in case of.

then `export const pageProps { i18ndata, nodeReleasesData }`

But again, I think one of the goals here was to **not** pass `nodeReleasesData` on the pageProps, and actually use a Hook. Have you done that?

Reply:
I think it is related to [this](https://github.com/nodejs/nodejs.org/pull/5365#discussion_r1193171159) comment.

Based on this comment (please correct me if I am wrong).
So, the fixture will be used by tests and not storybook?
Storybook will fetch real data instead.

Reply:
But again, @Harkunwar introduced on another PR a way to mock modules on Storybook.

I think you should follow his lead, and not pass the nodReleaseData as pageProps as I mentioned in a few places 🙇 (cc @Harkunwar if you could help @HinataKah0 to get into the right direction here 🙇)

Reply:
Should be in this [PR](https://github.com/nodejs/nodejs.org/pull/5378), right?
I am referring to these lines:
```
parameters: {
  moduleMock: {
    mock: () => {
      const mock = createMock(hooks, 'useNodeJsContributorsApi');
      mock.mockReturnValue({
        profileUri: 'https://github.com/nodejs',
        avatarUri: 'https://avatars.githubusercontent.com/u/9950313?s=75&v=4',
        login: 'nodejs',
        contributionsCount: 20,
        commitsListUri:
          'https://github.com/nodejs/node/commits?author=nodejs',
      });
    },
  },
},
```
It looks good to me! 👍

Reply:
Yup. If you could implement this on your tests, that'd be great. So we can remove the releaseData from pageProps :)

Reply:
@HinataKah0 LMK if you did implement this on your Storybook 👀